rename_images.py

```diff
@@ -0,0 +1,20 @@
+import os
+
+files = os.listdir('./data/')
+
+image_info = {}
+for f in files:
+    parts = os.path.splitext(f)[0].split('_')
+    if len(parts) <= 2:
+        continue
+    if parts[0] not in image_info.keys():
```

Comment:
Assign names to the parts before this line, otherwise the rest of the code is hard to read.